tasks/ci_test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+EXIT_STATE=0
+
+case $CIRCLE_NODE_INDEX in
+
+    0)
+        npm run test-image      || EXIT_STATE=$?
+        npm run test-image-gl2d || EXIT_STATE=$?
+        npm run test-export     || EXIT_STATE=$?
+        npm run test-syntax     || EXIT_STATE=$?
+        npm run lint            || EXIT_STATE=$?
+        exit $EXIT_STATE
```

Comment:
**Important:** I chose this `|| EXIT_STATE=$?` pattern over `#!/bin/bash -e` because I wanted all tests to run commands to run even if one of them fails. Note that `$?` is the exit code of the previously command. Stuff to the right of `||` is only executed if in the command to the left exits with a non-zero code.

Does anyone know a better to do this?